assets/scss/_variables_project.scss

```
@@ -0,0 +1,254 @@
+// Project-level Bootstrap/Docsy variable overrides for the JA handbook site.
+//
```

Stylelint (17.10.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)